definitions/npm/react-router-dom_v6.x.x/flow_v0.201.x-/react-router-dom_v6.x.x.js

```diff
@@ -424,9 +675,9 @@ declare module 'react-router-dom' {
   declare export function useLoaderData(): any;
 
   declare export type RouterProviderProps = {|
-    fallbackElement?: React$Node;
-    router: typeof Router;
-  |}
+    fallbackElement?: React$Node,
+    router: RemixRouter,
```

Comment:
I've spotted an issue with the misleading `Router` type/component

a «RemixRouter» is expected https://github.com/remix-run/react-router/blob/edf1dfc3327f721045ed672870b4f99825aa8fb8/packages/react-router/lib/components.tsx#LL54C23-L54C23